addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2049,19 +2049,23 @@ var SnippetsMenu = Widget.extend({
         }
 
         // Prepare the functions
-        var functions = {
-            is: function ($from) {
-                return $from.is(selector) && $from.filter(filterFunc).length !== 0;
-            },
-        };
+        var functions = {};
         if (noCheck) {
+            functions.is = function ($from) {
+                return $from.is(selector) && $from.filter(filterFunc).length !== 0;
+            };
             functions.closest = function ($from, parentNode) {
                 return $from.closest(selector, parentNode).filter(filterFunc);
             };
             functions.all = function ($from) {
                 return ($from ? dom.cssFind($from, selector) : $(selector)).filter(filterFunc);
             };
         } else {
+            functions.is = function ($from) {
+                return $from.is(selector) 
+                    && [...self.getEditableArea()].some(el => $(el).find($from).length || $(el).is($from))
```

Comment:
Seems strange and hard to read because you mix jQuery and not jQuery... actually worsening the perf of jQuery itself. Either only use jQuery (since old code) or not at all (like we should for new code). With jQuery you only need one call to `find` and one to `addBack`. Without jQuery, it's like you did but don't build `$(el)` (muttiple times), don't use `find`, don't `use` is.

Reply:
ok done :+1: